### PR TITLE
fix(transformer): fix 'pub build' in examples

### PR DESCRIPTION
**Pull request by sigmundch**

### CODE REVIEW

**Comment by sigmundch** [comment]
@yjbanov here is the patch

File: modules/examples/pubspec.yaml [hunk 1]
```diff
@@ -16,6 +16,9 @@ dependency_overrides:
     path: ../angular2_material
 transformers:
 - angular2:
+    # The build currently fails on material files because there is not yet
+    # support for transforming cross-package urls.
```
Comment (yjbanov) [inline modules/examples/pubspec.yaml]: Please log an issue and leave a TODO with a link to the issue.

Comment (sigmundch) [inline modules/examples/pubspec.yaml]: done

Comment (sigmundch) [inline modules/examples/pubspec.yaml]: (will upload new version in a sec)

**Comment by yjbanov** [comment]
Merged as https://github.com/angular/angular/commit/62589293aaa9efc6849e06c7f3629731699784de

**Pull request closed without merging**